action/remove.go

```diff
@@ -2,24 +2,70 @@ package action
 
 import (
 	"os"
+	"os/exec"
 	"path/filepath"
+	"strings"
 
 	"github.com/helm/helm/log"
+	"github.com/helm/helm/manifest"
 )
 
+// kubeGet executes the kubectl command, override in tests
+var kubeGet func(string) string
```

Comment:
+1 to adding the type.

I'd like to propose an alternative to the global implementation if I may. It seems that `kubeGet` is only used once in this file. I didn't check if it's used elsewhere but my proposal would stand for any other locations in code as well.

Instead using a global `kubeGetter`, my alternative is passing the `kubeGetter` into the `func`s that require it, and then passing the global implementation as needed from production code. Test code can then pass test implementations as needed.

For example, the global `func` is used [here](https://github.com/helm/helm/pull/257/files#diff-ac2075f1a23904bf5eb1df751e18eb6cR63). My suggesting is to add a `kubeGetter` parameter to the [`checkManifests`](https://github.com/helm/helm/pull/257/files#diff-ac2075f1a23904bf5eb1df751e18eb6cR53) func and having code pass the implementation manually, and have test code pass a test version.

This proposed change will make the code more explicit in what it's doing.